staging/src/k8s.io/apiserver/pkg/server/options/authorization.go

```diff
@@ -78,6 +78,14 @@ func NewDelegatingAuthorizationOptions() *DelegatingAuthorizationOptions {
 		DenyCacheTTL:        10 * time.Second,
 		ClientTimeout:       10 * time.Second,
 		WebhookRetryBackoff: DefaultAuthWebhookRetryBackoff(),
+		// This allows the kubelet to always get health and readiness without causing an authorization check.
+		// This field can be cleared by callers if they don't want this behavior.
+		AlwaysAllowPaths: []string{"/healthz", "readyz", "/livez"},
```

Comment:
I think we should at least require membership in the authenticated group for something like this. Does it? How?

Reply:
Assuming you have anonymous auth enabled on your cluster, we already allow this for unauthenticated users:

https://github.com/kubernetes/kubernetes/blob/7b3f0cd5ca47573478cf32a1101b2ef01ac23e72/plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go#L228-L236
https://github.com/kubernetes/kubernetes/blob/7b3f0cd5ca47573478cf32a1101b2ef01ac23e72/plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go#L578

This change just makes it so that the SAR checks are skipped.